### PR TITLE
poppler 21.11.0 rebuild for cairo 1.16.0 pinning

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Rebuild for cairo pinning to 1.16.0. 
- Remove s390x skip, as boost-cpp is now available for s390x.
- Updated metadata.

Dev: https://gitlab.freedesktop.org/poppler/poppler/-/tree/poppler-21.11.0
License: https://gitlab.freedesktop.org/poppler/poppler/-/blob/poppler-21.11.0/COPYING

Jira: https://anaconda.atlassian.net/browse/DSNC-4947

TODO: abs.yaml and conda_build_config.yaml to be removed before merging.

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — COMMENTED** [review]
I am wondering if we should use the python skip instead of directly stating the pinning for python.

https://github.com/AnacondaRecipes/poppler-feedstock/blob/cairo_rebuild/recipe/meta.yaml#L53

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channels: 
+  cbouss: cairo
```
Comment (tobijk) [inline abs.yaml]: Don't forget to remove this :)

**Review by tobijk — APPROVED** [review]

**Comment by cbouss** [comment]
Addressed comment from Maxwell by new commit.

However, the build now fails because since the last time, aggregate cbc was updated and there is now a dependency on libxcb which creates problems.

**Comment by cbouss** [comment]
Hmm now there is an intermittent issue when building poppler.
We sometimes get ModuleNotFoundError: No module named 'giscanner._giscanner' 

I cannot merge this as it is. I either have to disable introspection or find the root cause and correct it.

**Pull request closed without merging**